### PR TITLE
Topic 1 - Exercise 3

### PR DESCRIPTION
**Pull request by famecrew**

### CODE REVIEW

File: 00-HTML-CSS-basics/CSS/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                <div>Article</div>
+            </article>
+            <footer class="section-footer"><div>Footer</div></footer>
+        </section>
+        <aside class="sidebar"><div>Aside</div></aside>
+        <footer class="footer"><div>Footer</div></footer>
+
+
+
+        <script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline 00-HTML-CSS-basics/CSS/index.html]: Same as before...

File: 01-JS-DOM-APIs/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Topic 1</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```
Comment (DemianMoschin-GLB) [inline 01-JS-DOM-APIs/index.html]: +1

File: 01-JS-DOM-APIs/index.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+             Quisque ac sem a ex maximus elementum. Nullam gravida ultrices facilisis. Phasellus rhoncus lorem tincidunt, luctus est vitae, fermentum dui. Nulla imperdiet quis tortor eget fringilla. Nulla vel tincidunt enim. Cras at urna eget ipsum lobortis egestas. Quisque quam nunc, vehicula non metus quis, tempus ultricies dolor. Donec sodales nibh in vulputate commodo. Sed hendrerit volutpat lorem, auctor semper est faucibus vitae. Praesent vel eleifend urna, at maximus metus. Morbi eget urna a ante congue gravida. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam aliquet sit amet risus consectetur egestas. Ut fermentum venenatis purus eu aliquet.
+    </section>
+    <section>
+        <h2 class="fade-in" id="fadeIn">Hello world!</h2>
+    </section>
+    <button onclick="getConfig('GET','http://api.icndb.com/jokes/random',true)">Click me</button>
```
Comment (DemianMoschin-GLB) [inline 01-JS-DOM-APIs/index.html]: Please don't include function definitions inside this callback handler. 

Put it inside your js file!

File: 01-JS-DOM-APIs/script.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+function myFunction() {
+    document.getElementById("fadeIn").style.opacity = "1";
```
Comment (DemianMoschin-GLB) [inline 01-JS-DOM-APIs/script.js]: Could you give a more intuitive name for this ID?

Comment (famecrew) [reply 01-JS-DOM-APIs/script.js]: Done

File: 00-HTML-CSS-basics/CSS/index.html (hunk 1 above)
Comment (famecrew) [reply 00-HTML-CSS-basics/CSS/index.html]: Done

File: 01-JS-DOM-APIs/script.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+function getConfig() {
+    config = ['GET','http://api.icndb.com/jokes/random',true];
+    return onClick(config);
+}
+
+function onClick(config) {
+    const xhttp = new XMLHttpRequest();
+    xhttp.open(config[0],config[1],config[2]);
```
Comment (DemianMoschin-GLB) [inline 01-JS-DOM-APIs/script.js]: Could you use an object instead of an array here?

**Comment by DemianMoschin-GLB** [comment]
Looks great!

File: 01-JS-DOM-APIs/script.js (hunk 5 above)
Comment (famecrew) [reply 01-JS-DOM-APIs/script.js]: Done

**Pull request merged**